documentation/release_notes.rst

```diff
@@ -8,6 +8,85 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.8.0
+===============
+
+New Features
+------------
+- Added support of host policies for ``histogram`` algorithms.
+- Added support for specification of a size limit for the output range in the range-based ``merge`` algorithm.
+- Improved performance of the ``merge`` and sorting algorithms
+  (``sort``, ``stable_sort``, ``sort_by_key``, ``stable_sort_by_key``) that rely on Merge sort [#fnote1]_,
+  with device policies for large data sizes.
+- Improved performance of ``set_intersection``, ``set_union``, ``set_difference``, ``set_symmetric_difference``
+  algorithms with device policies.
+- Improved performance of ``reduce_by_segment`` with device policies for GPUs with sub-group sizes of 32.
```

Comment:
Should we just omit the part about subgroup sizes of 32?  Most all cards have this available, and users may not know enough to understand this.

Reply:
Yes, that's likely too much detail.

Reply:
Done.